test/integration/tasks.test.js

```diff
@@ -284,7 +283,7 @@ describe('Tasks', function () {
     it('Should return 404 when username is invalid', function (done) {
```

Comment:
Also add a test case with status `inactive`, `invalidstring`, blank, etc

Reply:
@ankurnarkhede  Is it ok if I add these test under [this](https://github.com/Real-Dev-Squad/website-backend/issues/381) task as under this task I have to write almost all test cases again, which will revamp whole code of tasks integration test.

Reply:
@ankurnarkhede Will do this change under [this](https://github.com/Real-Dev-Squad/website-backend/issues/381) task.